openmc/cell.py

```diff
@@ -139,7 +143,10 @@ def __repr__(self):
             string += '\t{0: <15}=\t{1}\n'.format('Temperature',
                                                   self.temperature)
         string += '{: <16}=\t{}\n'.format('\tTranslation', self.translation)
-        string += '{: <16}=\t{}\n'.format('\tVolume', self.volume)
+        if self._volume_std is not None:
+            string += '{: <16}=\t{} +/- {}\n'.format('\tVolume', self.volume, self.volume_std)
+        else:
+            string += '{: <16}=\t{}\n'.format('\tVolume', self.volume)
```

Comment:
I got mixed up with the other similar doc string for a second there. Now in [cm^3]

```suggestion
            string += '{: <16}=\t{} +/- {} [cm^3]\n'.format('\tVolume', self.volume, self.volume_std)
        else:
            string += '{: <16}=\t{} [cm^3]\n'.format('\tVolume', self.volume)
```